runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/BundleProgressHandler.java

```diff
@@ -34,7 +34,7 @@
   void onCompleted(ProcessBundleResponse response);
 
   /** Returns a handler that ignores metrics and throws on splits (as splits can not be ignored). */
-  static BundleProgressHandler unsupported() {
```

Comment:
I actually liked the term `unsupported` more then `defaultInstance`, I just wasn't clear as to why the tests weren't breaking all over the place and was just looking for an explanation. Should have been clearer in my intent.